src/rgw/rgw_crypt.cc

```diff
@@ -172,6 +172,74 @@ CryptoAccelRef get_crypto_accel(CephContext *cct)
 }
 
 
+#ifdef USE_OPENSSL
+template <std::size_t KeySizeV, std::size_t IvSizeV>
+static inline
+bool evp_sym_transform(CephContext* const cct,
+                       const EVP_CIPHER* const type,
+                       unsigned char* const out,
+                       const unsigned char* const in,
+                       const size_t size,
+                       const unsigned char* const iv,
+                       const unsigned char* const key,
+                       const bool encrypt)
+{
```

Comment:
is line continuation needed here?

Reply:
oh, it's not new anyway, sorry

Reply:
Do you mean  new line before the bracket? I tried to be consistent with closest neighborhood but I stay open to reformatting entire file.